cms/djangoapps/contentstore/management/commands/import_content_library.py

```diff
@@ -0,0 +1,111 @@
+"""
+Script for importing a content library from a tar.gz file
+"""
+from __future__ import print_function
+
+import base64
+import os
+import tarfile
+
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.exceptions import SuspiciousOperation
+from django.core.management.base import BaseCommand, CommandError
+from lxml import etree
+from opaque_keys.edx.keys import CourseKey
+from path import Path
+from xmodule.contentstore.django import contentstore
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import DuplicateCourseError
+from xmodule.modulestore.xml_importer import import_library_from_xml
+
+from cms.djangoapps.contentstore.utils import add_instructor
+from openedx.core.lib.extract_tar import safetar_extractall
+
+
+class Command(BaseCommand):
+    """
+    Import the specified content library archive.
+    """
+    help = 'Import the specified content library into mongo'
+
+    def add_arguments(self, parser):
+        parser.add_argument('archive_path')
+        parser.add_argument('owner_username')
+
+    def handle(self, *args, **options):
+        """
+        Given a content library archive path, import the corresponding course to mongo.
+        """
+
+        archive_path = options['archive_path']
+        username = options['owner_username']
+
+        data_root = Path(settings.GITHUB_REPO_ROOT)
+        subdir = base64.urlsafe_b64encode(os.path.basename(archive_path))
+        course_dir = data_root / subdir
+
+        # Extract library archive
+        tar_file = tarfile.open(archive_path)
+        try:
+            safetar_extractall(tar_file, course_dir.encode('utf-8'))
+        except SuspiciousOperation as exc:
+            raise CommandError(u'\n=== Course import %s: Unsafe tar file - %s\n', archive_path, exc.args[0])
```

Comment:
`CommandError` doesn't take format arguments as additional parameters - this should use `.format()` instead.

Reply:
👍